extensions/azurecore/package.nls.json

```diff
@@ -31,6 +31,7 @@
 	"config.azureDeviceCodeMethod": "Device Code Method",
 	"config.noSystemKeychain": "Disable system keychain integration. Credentials will be stored in a flat file in the user's home directory.",
 	"config.piiLogging": "Should Personally Identifiable Information (PII) be logged in the Azure Accounts output channel and the output channel log file.",
-	"config.loggingLevel": "[Optional] The verbosity of logging for the Azure Accounts extension."
+	"config.loggingLevel": "[Optional] The verbosity of logging for the Azure Accounts extension.",
+	"config.authenticationLibrary": "The library used for the AAD auth flow. Please restart ADS after changing this option."
```

Comment:
If this is something that does actually require a restart then we should be prompting/requiring the user to reload when the value is changed